debian/rules

```diff
@@ -134,7 +133,7 @@ configure_flags_udeb = \
 	--disable-xrandr
 
 %:
-	dh $@ --with gir
+	dh $@ --with gir,quilt
```

Comment:
Do we really need this bit? I don't see it in the `debian/rules` from GTK 3.22.29-2 in buster, and besides we don't really use quilt to patch GTK+ anyway.

Reply:
Yeah, we do. We actually do use quilt here because there's a patch in debian/patches (https://github.com/endlessm/gtk/blob/debian-master/debian/patches/071_fix-installation-of-HTML-images.patch) that we need in OBS but not on jenkins. At least, we used to need it. That's what the `include /usr/share/cdbs/1/rules/patchsys-quilt.mk` did in the cdbs world. We could analyze if that patch is still needed or if we can put it on our main branch, but this just tries to keep the existing setup working.

The reason why we need to pull in quilt manually is because jenkins makes old school `1.0` format source packages and not `3.0 (quilt)` packages. The reason for that is jenkins needs to be generic enough to handle any debian packaging style we throw at it. I think there are ways to make this better (I think I did it for the `jenkins-infra` flow), but that's the way it works now.

Reply:
Ah, I see. It makes sense then now, I'll merge this. Thanks!